index.yaml

```diff
@@ -1,3 +1,5 @@
+---
+
```

Comment:
Can we remove the requirement for this with https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.document_start?

Reply:
Yes, I never was a fan of that either.